### PR TITLE
Docs:  Updates to the Query management in Explore page

### PR DESCRIPTION
**Pull request by lwandz13**
Updates for clarity and also to reflect UI changes.

### CODE REVIEW

File: docs/sources/explore/query-management.md [hunk 1]
```diff
@@ -8,77 +8,88 @@ labels:
...
+{{% /admonition %}}
 
 ## Query history
 
-Query history is a list of queries that you used in Explore. The history is stored in the Grafana database and it is not shared with other users. The retention period for queries in history is two weeks. Queries older than two weeks are automatically deleted. To open and interact with your history, click the **Query history** button in Explore.
+Query history contains the list of queries that you created in Explore. This history is stored in the Grafana database and isn't shared with other users. The retention period for a query history is **two weeks**. Queries older than two weeks are automatically deleted. 
 
 {{% admonition type="note" %}}
-Starred queries are not subject to the two weeks retention period and they are not deleted.
+Starred queries aren't subject to the two week retention period and aren't deleted.
```
Comment (chri2547) [inline docs/sources/explore/query-management.md:28]:
```suggestion
Starred queries aren't subject to the two-week retention period and aren't deleted.
```

File: docs/sources/explore/query-management.md [hunk 2]
```diff
@@ -8,77 +8,88 @@ labels:
...
 {{% /admonition %}}
 
-### View query history
+To view your query history:
+
+1. Go to the Explore page
```
Comment (chri2547) [inline docs/sources/explore/query-management.md:33]:
```suggestion
1. Go to the Explore page.
```

File: docs/sources/explore/query-management.md [hunk 3]
```diff
@@ -8,77 +8,88 @@ labels:
...
-### Manage favorite queries
-
-All queries that have been starred in the Query history tab are displayed in the Starred tab. This allows you to access your favorite queries faster and to reuse these queries without typing them from scratch.
+By default, query history shows you newest queries first. Click the sort box in the upper right to change to **Oldest first** to older queries first.  You can search your queries using keywords. 
 
-### Sort query history
+### Query history Starred tab
 
-By default, query history shows you the most recent queries. You can sort your history by date or by data source name in ascending or descending order.
+All starred query are displayed in the Starred tab. This gives quick access to key or favorite queries without having to rewrite them.
```
Comment (chri2547) [inline docs/sources/explore/query-management.md:57]:
```suggestion
All starred queries are displayed in the Starred tab. This gives quick access to key or favorite queries without having to rewrite them.
```

File: docs/sources/explore/query-management.md [hunk 4]
```diff
@@ -8,77 +8,88 @@ labels:
...
-You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
+1. Click in the **Search queries** field.
+1. Type the keyword(s) or term you are want to search for in search field.
 
-1. Click the **Search queries** field.
-1. Type the term you are searching for into search field.
+### Query history Settings tab
 
-### Query history settings
+You can customize your query history in the Settings tab.
```
Comment (chri2547) [inline docs/sources/explore/query-management.md:86]:
```suggestion
You can customize your query history in the **Settings** tab.
```

**Review by chri2547 — APPROVED** [review]
A few suggestions. Thanks!

**Comment by lwandz13** [comment]
Thank you @chri2547!!

**Review by harisrozajac — APPROVED** [review]
🚀

**Pull request merged**